src/main/java/seedu/address/logic/commands/FindTagCommand.java

```diff
@@ -0,0 +1,37 @@
+package seedu.address.logic.commands;
+
+import seedu.address.model.person.TagsContainKeywordsPredicate;
+
+/**
+ * Finds and lists all persons in address book whose tags contains any of the argument keywords.
+ * Keyword matching is case insensitive.
+ */
+public class FindTagCommand extends Command {
+
+    public static final String COMMAND_WORD = "findtag";
+    public static final String COMMAND_ALIAS = "ft";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose tags contain any of "
+            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
+            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
+            + "Example: " + COMMAND_WORD + " friends colleagues neighbours";
+
+    private final TagsContainKeywordsPredicate predicate;
```

Comment:
Perhaps the variable name could be more descriptive? Something like
`tagsContainKeywordsPredicate`